index.es.js

```diff
@@ -1,12 +1,5 @@
-export {
-  Future,
-  Future as default,
-  isFuture,
-  reject,
-  of,
-  never,
-  isNever
-} from './src/core';
+export {default, default as Future} from './src/future';
```

Comment:
I've made the same mistake in other places of the code. I'll merge this PR as is, and create a separate issue to discuss the matter.